docs/topics/crud-mission-design-tradeoffs.adoc

```diff
@@ -2,8 +2,8 @@
 [width="100%",options="header"]
 |====================================================================
 |Pros           |Cons
-a| * Each runtime decides how the database interactions are implemented. One can use JDBC while others can use JPA or access ORM APIs directly. Each runtime decides what would be the best way.
+a| * Each runtime determines how  to implement the database interactions. One can use JDBC while others can use JPA or access ORM APIs directly. Each runtime decides what would be the best way.
```

Comment:
Really not sure why "determine" is better than "decide". IMHO, the word "decide" carries a lot less baggage.

Reply:
Fine with me, you're the wordsmiths here :-)

Reply:
Great I agree.